### PR TITLE
Fix error in activity and sequence diagram for DG

### PR DESCRIPTION
**Pull request by limjunxian1**

### CODE REVIEW

File: docs/diagrams/AddAltSequenceDiagram.puml
Comment (lordidiot) [inline docs/diagrams/AddAltSequenceDiagram.puml:1]: It's not clear when `createAddAltPerson` ends in the diagram, can you add the activation bar accordingly?

**Review by lordidiot — CHANGES_REQUESTED** [review]
Looks good overall, just one thing to fix!

File: docs/diagrams/AddAltSequenceDiagram.puml
Comment (limjunxian1) [reply docs/diagrams/AddAltSequenceDiagram.puml:1]: yes sure! fixed in 7f38c91e

**Review by lordidiot — APPROVED** [review]
LGTM

**Pull request merged**